tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -233,6 +233,42 @@ testsToRun.forEach(test => {
                     ], done);
                 });
             });
+
+            describe('with versioning suspended', () => {
+                beforeEach(done => series([
+                    cb => cloudServer.createVersionedBucket(srcBucket, cb),
+                ], done));
```

Comment:
unless we expect to add more steps later

```suggestion
                beforeEach(done => cloudServer.createVersionedBucket(srcBucket, done));
```